tasks/build.py

```diff
@@ -161,12 +161,31 @@ def build_easystack_from_pr(pr, event_info):
 
         # enlist jobs to proceed
         jobs.append([arch_job_dir,arch_target,slurm_opt])
-
-
     log("  %d jobs to proceed after applying white list" % len(jobs))
-    if len(jobs) > 0:
+    if jobs:
         log(json.dumps(jobs, indent=4))
+    
+    return repo_name, jobs
+
 
+def build_easystack_from_pr(pr, event_info):
+    # retrieving some settings from 'app.cfg' in bot directory
+    # [github]
+    app_name = config.get_section('github').get('app_name')
+    
+    # [buildenv]
+    jobs_base_dir, local_tmp, build_job_script, submit_command, slurm_params, cvmfs_customizations, http_proxy, https_proxy, load_modules = get_build_env()
```

Comment:
maybe this approach is better:

```python
build_env_cfg = get_build_env_configuration()
```
and use constants:
```python
SUBMIT_COMMAND = 'submit_command'
```
in `get_build_env_configuration`:
```python
    buildenv = config.get_section('buildenv')
    ...
    submit_command = buildenv.get(SUBMIT_COMMAND)
```

where you need the submit command, use:
```python
build_env_cfg[SUBMIT_COMMAND]
```
and pass down `build_env_cfg`:
```python
    repo_name, jobs = download_pull_request(pr, arch_target_map, run_dir, build_env_cfg)
    for job in jobs:
        submit_job(job, build_env_cfg)